app/packages/core/src/plugins/SchemaIO/components/NativeModelEvaluationView/Error.tsx

```
@@ -0,0 +1,40 @@
+import { West } from "@mui/icons-material";
+import { Box, Button, Card, Stack, Typography } from "@mui/material";
+import React from "react";
+import ErrorIcon from "./ErrorIcon";
+
+export default function Error(props: ErrorProps) {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)